### ISSUE TITLE
Consider also checking for last commit when bumping stale PR

### ISSUE THREAD

**Issue by pjcozzi** [AnalyticalGraphicsInc/cesium-concierge#60]
Expand on #57 if it turns out to be a problem.

**Comment by OmarShehata** [comment]
Yes! It is in fact the next thing I'll do after https://github.com/AnalyticalGraphicsInc/cesium-concierge/pull/118 since we just ran into an instance where Concierge bumped something that was actively being developed.
